distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfigStatusCommand.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.test.itests.platform;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+
+import ddf.common.test.BeforeExam;
+import ddf.common.test.KarafConsole;
+import ddf.test.itests.AbstractIntegrationTest;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class TestConfigStatusCommand extends AbstractIntegrationTest {
+
+    private static final String PLATFORM_CONFIG_STATUS_COMMAND = "platform:configstatus";
+
+    private static final String SUCCESSFUL_IMPORT_MESSAGE = "All config files in [%s] imported successfully.";
+
+    private static final String FAILED_IMPORT_MESSAGE = "Failed to import file [%s]. Moved to [%s].";
+
+    private static final String INVALID_CONFIG_FILE_1 = "ddf.test.itests.platform.TestPlatform.invalid.config";
+
+    private static final String INVALID_CONFIG_FILE_2 = "ddf.test.itests.platform.TestPlatform.startup.invalid.config";
+
+    private static KarafConsole console;
+
+    @BeforeExam
+    public void beforeExam() throws Exception {
+        basePort = getBasePort();
+        getAdminConfig().setLogLevels();
+        getServiceManager().waitForAllBundles();
+        console = new KarafConsole(bundleCtx);
+    }
+
+    @Test
+    public void testConfigStatusImportSuccessful() {
+        Path configDirectory = getPathToConfigDirectory();
+        String output = console.runCommand(PLATFORM_CONFIG_STATUS_COMMAND);
+        assertThat(output,
+                containsString(String.format(SUCCESSFUL_IMPORT_MESSAGE, configDirectory)));
+    }
+
+    @Test
+    public void testConfigStatusFailedImports() throws Exception {
```

Comment:
Do you think this test is a little too "whitebox", i.e., it assumes that the list of files is based on the content of the failed directory? Could we drop in the two invalid files like we do in the `TestPlatform` tests instead of manually copying them to the failed directory?